content/language/nodejs/develop.md

```diff
@@ -148,7 +148,7 @@ development, you can use one multi-stage Dockerfile for both.
 
 Update your Dockerfile to the following multi-stage Dockerfile.
 
-```dockerfile
+```dockerfile {hl_lines="5-27"}
 # syntax=docker/dockerfile:1
 
 ARG NODE_VERSION=18.0.0
```

Comment:
nit (non-blocking): line 170 sets `NODE_ENV` to `production`, and the `npm ci` command on line 174 has `--omit=dev`. Having both of these is redundant. Same on the `run-tests.md` page.